### ISSUE TITLE
Dataset size and learning rate

### ISSUE THREAD

**Issue by denisvorotyntsevbidease** [catboost/catboost#2571]
Problem:

I train a CatBoost model using the training part of the dataset to estimate the validation error. Then, I retrain the model using the full data (training + validation). Do I need to change the hyperparameters of CatBoost to improve performance (number of trees, learning rate, etc.)? 

I heard a recommendation to increase the learning rate by the square root of the increased sample size (e.g., 30% more data -> increase the learning rate by 1.3^0.5), but I haven't found any theoretical justifications yet. Any recommendations? Is it a heuristic? 

catboost version:
Operating System:
CPU:
GPU:
